Binstat error `panic: ERROR: BINSTAT: INTERNAL: t1=90173378966800 but t2=90173378966800`

## 🐞 Bug Report

A clear and concise description of what the bug is.

## What is the severity of this bug?

Important

## Reproduction steps

This bug is hard to reproduce because behaviour is different on different machines.  Ultimately, this error is throwing in the Flow CLI in a user report on Discord, but I cannot reproduce it myself.

```
PS C:\Users\NtTestAlert> flow-c1
panic: ERROR: BINSTAT: INTERNAL: t1=90173378966800 but t2=90173378966800


goroutine 1 [running]:
github.com/onflow/flow-go/utils/binstat.init.0()
        /root/go/pkg/mod/github.com/onflow/flow-go@v0.33.2-0.20240306234901-64ab8d27ea30/utils/binstat/binstat.go:246 +0xefa
```

To me, it looks like this could possibly be related to the time interval being measured being too small in comparison to whatever precision is available to "nanosecond" times on the machine (but this is just my hunch)?  It could be something entirely unrelated, and maybe this value is entirely invalid?

## Expected behaviour

This error should not appear

## Screenshots

n/a

## Specifications

- System: Windows
- Flow CLI: v1.15.0-cadence-v1.0.0-preview.13
- Architecture: unknown

## Additional context

https://discordapp.com/channels/613813861610684416/1221882991832203304/1221888889619222578

--- a/utils/binstat/binstat.go
+++ b/utils/binstat/binstat.go
@@ -242,7 +242,7 @@ func init() {
 	}
 
 	t2 := runtimeNanoAsTimeDuration()
-	if t2 <= t1 {
+	if t2 < t1 {
 		panic(fmt.Sprintf("ERROR: BINSTAT: INTERNAL: t1=%d but t2=%d\n", t1, t2))
 	}
 }

